### DISCUSSION TITLE
Multiple cells per gNodeB - handover srsUE zeroMQ

### DISCUSSION THREAD

**Discussion by P4sca1** [srsran/srsRAN_Project#575] [General Help] [answered]
Hello 👋

congratulations on the 24.4 release! 🥳
I would like to test intra-gNodeB handover using srsUE with a simulated radio channel (zeroMQ).
Therefore I have the following questions:
1. How to setup multiple cells for one gNodeB? I checked the configuration reference and the cells configuration seems to be the correct starting point. However, it refers to a file called gnb_custom_cell_properties.yml, which I was not able to find.
2. How to initiate a handover with srsUE? Is this possible without a graphical user interface like gnuradio?

**Comment by pgawlowicz** [comment, marked as answer]
Hi, unfortunately, srsUE does not support HO.

**Reply by P4sca1** [reply]
Is this something that is only supported for 4G? I found the following tutorial about Intra eNodeB handover and would like to do the same with one gNodeB hosting 2 cells.
https://docs.srsran.com/projects/4g/en/hoverxref_test/app_notes/source/handover/source/index.html

**Reply by pgawlowicz** [reply]
yes, this is only for 4g
